domador.js

```diff
@@ -150,7 +150,8 @@ Domador.prototype.append = function append (text) {
 };
 
 Domador.prototype.br = function br () {
-  this.append('  ' +  this.left);
+  if (!this.inTable) this.append('  ' +  this.left);
+  else this.append('  <br />');
```

Comment:
I started here, and it results in cleaner `<br />`s, but I have an alternative version which does not add these two lines, and processes `<br />`s the same way as other block elements. That version is simpler and single-track, but won't clean up a messy `bar<br>` into `bar <br />` the way this will. Preference?